src/game/Spells/SpellMgr.cpp

```diff
@@ -132,6 +132,9 @@ DiminishingGroup GetDiminishingReturnsGroupForSpell(SpellInfo const* spellproto,
             // Faerie Fire
             else if (spellproto->SpellFamilyFlags[0] & 0x400)
                 return DIMINISHING_LIMITONLY;
+            // Feral Charge Root Effect
+            else if (spellproto->Id == 45334)
```

Comment:
it would be better to wrap those numbers into costants

Reply:
Give an example because if someone is porting from TC, it might not be obvious.